src/Microsoft.DotNet.Interactive.AspNetCore/AspNetCoreCSharpKernelExtensions.cs

```diff
@@ -37,18 +38,20 @@ public static class AspNetCoreCSharpKernelExtensions
         typeof(IEndpointRouteBuilder).Namespace, // Microsoft.AspNetCore.Routing
         typeof(EndpointRouteBuilderExtensions).Namespace, // Microsoft.AspNetCore.Builder
         typeof(InteractiveEndpointRouteBuilderExtensions).Namespace, // Microsoft.DotNet.Interactive.AspNetCore
-        typeof(HttpClient).Namespace, // System.Net.Htttp
+        typeof(HttpClient).Namespace, // System.Net.Http
     };
 
     public static CSharpKernel UseAspNetCore(this CSharpKernel kernel)
     {
         InteractiveHost interactiveHost = null;
 
+        HttpResponseMessageFormattingExtensions.RegisterFormatters();
```

Comment:
@jonsequitur Now that I think about it, this may not be a good idea since this could override formatters that were registered by the user after `CommandLineParser.SetUpFormatters` was called in the tool.

Perhaps this registration should happen directly in the tests (which depend on the formatter)... I'll change this.